### PR TITLE
Add survey sec bugs

### PR DESCRIPTION
**Pull request by mozfreddyb**

### CODE REVIEW

File: auto_nag/scripts/survey_sec_bugs.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+    def __init__(self):
+        super(SurveySecurityBugs, self).__init__()
+        self.changes_per_bug = {}
+
+    def description(self):
+        return "Submit survey to assignee of a security bug"
+
+    def get_bz_params(self, date):
+        params = {
```
Comment (calixteman) [inline auto_nag/scripts/survey_sec_bugs.py]: nit: sort the f,o,v fields, I'd put chfield... after resolution and then f1,o1,v1,f2,o2,v2,....

File: auto_nag/scripts/survey_sec_bugs.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag.bzcleaner import BzCleaner
+
+
+class SurveySecurityBugs(BzCleaner):
+    LIST_OF_PEOPLE_TO_REACH_OUT = [
```
Comment (calixteman) [inline auto_nag/scripts/survey_sec_bugs.py]: nit: put that stuff in tools.json file
```json
"survey_sec_bugs": { "to_reach_out": ["foo", "bar", ...]}
```
and then just use `self.get_config("to_reach_out", default=[])` when needed in the python code

File: auto_nag/scripts/survey_sec_bugs.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        }
+
+        return bug
+
+    def get_autofix_change(self):
+        return self.changes_per_bug
+
+    def comment_tpl_for_bugid(self, bugid):
+        URL = (
+            "https://docs.google.com/forms/d/e/1FAIpQLSe9uRXuoMK6tRglbNL5fpXbun_oEb6_xC2zpuE_CKA_GUjrvA/viewform"
```
Comment (calixteman) [inline auto_nag/scripts/survey_sec_bugs.py]: nit: use a f-string

File: auto_nag/scripts/survey_sec_bugs.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+            "?usp=pp_url&entry.2124261401="
+            + "https%3A%2F%2Fbugzilla.mozilla.org%2Fshow_bug.cgi%3Fid%3D"
+            + bugid
+        )
+
+        return (
+            "As part of a security bug pattern analysis, we are requesting your help with a high level analysis"
```
Comment (calixteman) [inline auto_nag/scripts/survey_sec_bugs.py]: nit: f-string too

File: auto_nag/scripts/survey_sec_bugs.py (hunk 1 above)
Comment (mozfreddyb) [reply auto_nag/scripts/survey_sec_bugs.py]: Would you mind if I leave them unsorted? Sorting them like field/operator/value makes the query readable in a way that the query is reviewable IMHO.

**Pull request merged**